subtype.go

```diff
@@ -16,16 +16,16 @@ func (s *Session) GetAllShowSubtypes() (subtypes []ShowSeasonSubtype, err error)
 
 // GetShowSubtypeByClass returns a ShowSeasonSubtype based on a given class
 // Can return nil if no subtype found for given class
-func (s *Session) GetShowSubtypeByClass(class string) (subtype ShowSeasonSubtype, err error) {
+func (s *Session) GetShowSubtypeByClass(class string) (ShowSeasonSubtype, error) {
```

Comment:
```suggestion
func (s *Session) GetShowSubtypeByClass(class string) (*ShowSeasonSubtype, error) {
```

that'll let you return nil